README.md

```diff
@@ -22,116 +22,223 @@ Issues should be reported via the forum here: https://forum.libreelec.tv/board/4
 
 # Windows
 
-### 1. install "Qt Online Installer"
-select: Custom install -> select QT 6.6.2 -> disable all besides MinGW
-Install to `C:\QT`
+## Install pre-requisites
 
-### 2. Install 7zip
+### 1. Install 7zip
 https://www.7-zip.org/download.html
 
-### 3. Install Git
+### 2. Install Git
 https://git-scm.com/download/win
 
-### 4. Install Ruby
-https://rubyinstaller.org/downloads/ select "WITHOUT DEVKIT" for download
+### 3. Install CMake
+https://cmake.org/download, select windows x64 installer and "Add cmake to PATH".
 
-### 5. Install Perl
-https://www.activestate.com/products/perl/ (needs registration) or
-
-https://www.softpedia.com/get/Programming/Coding-languages-Compilers/ActivePerl.shtml
-
-### 6. Install Python
+### 4. Install Python
 https://www.python.org/downloads/windows/ select Windows installer (64-bit) for download
 at install select "Add Python to PATH"
 
-### 7. Add programs to PATH variable
+#### 5. Add 7-Zip to PATH variable
+
+`C:\Program Files\7-Zip`
 
-`C:\Qt\Tools\Ninja`
-`C:\Qt\Tools\CMake_64\bin`
+## Building for Windows using MSys2 and MinGW
 
-reboot Windows
+### 1. Install msys2 and MinGW
 
-### 8. Clone Git Repo
-Clone the repository to `C:\usb-sd-creator`
+Install the latest msys2 via the installer (not the base installer): https://repo.msys2.org/distrib/x86_64/
+
+After the install, from the resulting msys2 console install mingw:
```

Comment:
you need to full close/restart mingw (after install) or the build will not work properly, I ran into that problem earlier

Reply:
Strange, I didn't, but no harm to add it just incase.

Reply:
By any chance do you just need to a new command prompt to pick up the PATH?

Reply:
possible, but the issues were no obviously path related - it was rather strange and I could reproduce it
I am not 100% sure but (from chat logs) i ran several times into that.

```
[31/43] Running rcc for resource LibreELEC_USB-SD_Creator_translations
FAILED: .rcc/qrc_LibreELEC_USB-SD_Creator_translations.cpp /d/GIT/usb-sd-creator/build/.rcc/qrc_LibreELEC_USB-SD_Creator_translations.cpp
/bin/sh CMakeFiles/qrc_LibreELEC_USB-SD_Creator_translations.cpp-7d2c366.sh 1294df003dfec724
RCC: Error in 'D:/GIT/usb-sd-creator/build/.rcc/LibreELEC_USB-SD_Creator_translations.qrc': Cannot find file '/d/GIT/usb-sd-creator/build/lang-ast_ES.qm'
[32/43] Automatic RCC for resources.qrc
ninja: build stopped: subcommand failed.
```